### ISSUE TITLE
Ranging monitoring not working on Android 7.0 

### ISSUE THREAD

**Issue by sebasecheverry** [Estimote/Android-SDK#283]
## Prerequisites

* [X ] Put an X between the brackets on this line if you have done all of the following:
    * My beacons have Estimote Monitoring enabled (Check it in the Estimote Cloud, or via app)
    * My Android device/devices supports BLE and has Android OS version >= 5.0.0 
    * My Android device/devices have bluetooth enabled
    * My app has Location Permissions granted
    
## Basic information

**Estimote SDK version:** compile 'com.estimote:proximity-sdk:0.4.1'
compile 'com.estimote:sdk:1.0.12'
gardle version 2.8

**Android devices affected:** [Paste here your Android device model/models that you are having problems with.]
Asus zenfone Max Plus.
Asus_X018D ZB5770TL

**Android OS version affected:** Android 7.0 nougat

**Beacon hardware version:** [Paste here your beacons hardware version (you can check that in Estimote Cloud).] 

Hardware Version
F3.3

## Description

ranging and regions dont work.  In older devices and versions of android it works perfectly but in my version it just doesnt work.

1. Try any ranging. It dont work. Maybe is my configuration and im doing something bad. but is strange since it works on older versions of android correctly.

it never enters: **beaconManager.setRangingListener** or **onEnteredRegion**

`
apply plugin: 'com.android.application'

android {
    compileSdkVersion 27
    buildToolsVersion "27.0.2"

    defaultConfig {
        applicationId "com.cyclops.pruebaestimote4"
        minSdkVersion 21
        targetSdkVersion 27
        versionCode 4
        versionName "1.3"
        useLibrary 'org.apache.http.legacy'
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    testCompile 'junit:junit:4.12'

    // add the following line, and replace "1.0.12" with the latest version
    // of Estimote Android SDK; you'll find the latest version number on:
    //   https://github.com/Estimote/Android-SDK/releases
    compile 'com.android.support:appcompat-v7:27.0.2'
    compile 'com.android.support:design:27.0.2'
    //compile 'com.estimote:proximity-sdk:0.4.1'
    compile 'com.estimote:sdk:1.0.12'
    compile 'com.android.support:support-v4:27.0.0'
    compile 'org.apache.httpcomponents:httpcore:4.4.1'
    compile 'org.apache.httpcomponents:httpclient:4.5'
}`

`// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:1.5.0'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        jcenter()
        maven {
            url "https://maven.google.com"
        }
    }
}

task clean(type: Delete) {
    delete rootProject.buildDir
}
`

**Expected behavior:** I expect ranging to work correctly in newer versions of android. It works on older versions

**Actual behavior:** it works once every 2 hours and badly. (not all beacons detected)

## Additional information
```
Add here any logs you want to attach, or any other info, like pictures for example
```

**Comment by Poberro** [comment]
I see that you included SDK in version 1.0.12:
`compile 'com.estimote:sdk:1.0.12'`
but latest version is `1.4.1`. Can you try that version? 
As release log says there were some fixes for Android 7.
If it does not work can you provide some logcat or  [BT Snoop logs](https://blog.bluetooth.com/debugging-bluetooth-with-an-android-app)?

**Comment by Poberro** [comment]
No answer from author. Closing.
Ranging and Monitoring is deprecated. Please use [Proximity SDK](https://github.com/Estimote/Android-Proximity-SDK) instead.
